pkg/internal/apply/applier_integration_test.go

```diff
@@ -8,40 +8,38 @@ import (
 	"os"
```

Comment:
I tried to run the integration tests using go test -v -tags=integration but got "unable to retrieve AddonsLayer 'test'"
Suggest we create a dev guide for the project like https://github.com/paulcarlton-ww/go-utils/blob/master/docs/dev-guide.md to explain how to buil, test and run etc.
Created issue for this work https://github.com/fidelity/kraan/issues/29